### DISCUSSION TITLE
Welcome to KeepYourMouthShut Discussions!

### DISCUSSION THREAD

**Discussion by rajtilakjee** [rajtilakjee/keepyourmouthshut#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by drjagan** [comment]
Hi, I am Jagan Mohan. 
Keepyourmouthshut seems to be a promising tool. 
However, the site seems to be down. 
Hope this project is still live, it is unique and if paired with castopod, can be fantastic.
